src/WorldCup/Game.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Cheba\PhpUnit\WorldCup;
+
+class Game
+{
+    public $homeTeam;
+    public $awayTeam;
+    public $homeScore;
```

Comment:
Don't use public that way. It could be for [DTO](https://stackoverflow.com/questions/1051182/what-is-a-data-transfer-object-dto#1058186), but that code is not DTO. Also, allow setting score to "MyTeam" and setting homeTeam to -11.

Use also [type declaration](https://www.php.net/manual/en/language.types.declarations.php)

Reply:
Changed type declaration to private.